components/checks/passive/common_admin_interfaces.rb

```diff
@@ -0,0 +1,56 @@
+=begin
+    Copyright 2010-2014 Tasos Laskos <tasos.laskos@arachni-scanner.com>
+
+    This file is part of the Arachni Framework project and is subject to
+    redistribution and commercial restrictions. Please see the Arachni Framework
+    web site for more information on licensing and terms of use.
+=end
+
+# Looks for common administration interfaces on the server.
+#
+# @author Brendan Coles <bcoles@gmail.com>
+#
+# @version 0.1
+class Arachni::Checks::CommonAdminInterfaces < Arachni::Check::Base
+
+    def self.filenames
+        @filenames ||= read_file( 'admin-panels.txt' ) # wfuzz - wordlist/general/admin-panels.txt
```

Comment:
This may be problematic, [wfuzz](https://github.com/xmendez/wfuzz/) is licensed under the [GPLv2](https://github.com/xmendez/wfuzz/blob/master/LICENSE).

Have you cleared this with them?